server.js

```diff
@@ -1,10 +1,28 @@
 const express = require("express");
+const cors = require('cors')
+const pino = require('express-pino-logger')();
+const axios = require('axios')
+require('dotenv').config()
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
+app.use(cors())
 app.use(express.static('dist'));
+app.use(express.urlencoded({ extended: false }));
+app.use(pino);
+
+app.get('/captcha/validate_captcha', (req, res) => {
+    let token = req.query.token
+    axios.post("https://www.google.com/recaptcha/api/siteverify?secret=" + process.env.REACT_APP_CAPTCHA_SECRET_KEY + "&response=" + token)
```

Comment:
You can use Javascript template literals here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals